specification/eql/_types/EqlHits.ts

```diff
@@ -36,3 +37,23 @@ export class EqlHits<TEvent> {
    */
   sequences?: EqlHitsSequence<TEvent>[]
 }
+
+export class EqlHitsEvent<TEvent> {
```

Comment:
This type and the following one can just be called `HitsEvent` and `HitsSequence`.
```suggestion
export class HitsEvent<TEvent> {
```